apps/src/templates/ContentContainer.jsx

```diff
@@ -148,15 +148,17 @@ const styles = {
     display: 'inline',
     position: 'absolute',
     bottom: 20,
-    right: 0
+    right: 0,
+    lineHeight: 26
```

Comment:
I'm wondering if we should be saving some of these values as constants in CSS for consistency - are we using these kinds of values in multiple files?